### PR TITLE
Changes to FAQ page

### LINKED ISSUE #51: Update FAQ content
The language on the FAQ still has references to reporting status colors that aren't used in core Open SDG. https://github.com/open-sdg/open-sdg-site-starter/blob/develop/_pages/faq.md

EDIT (Lucy):

- [x] Update reporting status question with all colours ('Reported online' changed to 'Reported' and add 'Not applicable')
- [x] Reorder questions so order matched UK (suggested by UR as order that makes sense)
- [x] Add table of contents
- [x] Add question about tiers - copy from UK but add note about Tier 3 indicators being removed as part of comp review
- [x] Make existing questions more general
- [x] Are there still issues with IE8? if so, are we working to fix them? (see What browsers can I use to view this website?)

### PR DESCRIPTION
**Pull request by LucyGwilliamAdmin**
Fixes #51

### CODE REVIEW

File: _pages/faq.md [hunk 1]
```diff
@@ -5,31 +5,64 @@ permalink: /faq/
...
+- **Exploring data sources (white):** We are still looking for a suitable data source for this indicator.
+
+- **Not applicable (striped):** This indicator is not applicable in the national context.
+
+Where there is additional information about the status of indicator data collection and reporting, this will be displayed at the top of the indicator page.
+
+## Can other countries copy this website?
+Yes. This site is built and hosted using 100% free software and services, so other countries and localities can freely and easily reuse our code. Technical guidance on implementing this platform is available [here](https://open-sdg.readthedocs.io). Countries and localities exploring how to report their own SDGs data can also get support from the [Center for Open Data Enterprise (CODE)](http://www.opendataenterprise.org/) through their [SDG National Reporting Initiative](https://www.sdgreporting.org/).
+
 ## How accessible is this website?
```
Comment (LucyGwilliamAdmin) [inline _pages/faq.md:52]: @AnnCorp @brockfanning wondering if we need to change the answer to this question so it says something more like 

Open SDG aims to be as accessible and usable as possible for every user and constantly works towards meeting AA level of  [Web Content Accessibility Guidelines (WCAG 2.0)](https://www.gov.uk/service-manual/helping-people-to-use-your-service/understanding-wcag-20). However...

I'm just thinking if a country takes a copy of Open SDG and doesn't update the FAQ page but makes customisation, the answer as it is may be misleading

File: _pages/faq.md [hunk 2]
```diff
@@ -5,31 +5,64 @@ permalink: /faq/
...
+
+- [Can other countries copy this website?](#can-other-countries-copy-this-website)
+
+- [How accessible is this website?](#how-accessible-is-this-website)
+
+- [What browsers can I use to view this website?](#what-browsers-can-i-use-to-view-this-website)
+
+- [What do each of the three tiers mean?](#what-do-each-of-the-three-tiers-mean)
 
 ## How often will new data be added to this site?
```
Comment (LucyGwilliamAdmin) [inline _pages/faq.md:25]: @brockfanning @AnnCorp should we remove the "in cooperation with topic experts bit" - just thinking that may be misleading if countries aren't working with topic experts

**Comment by LucyGwilliamAdmin** [comment]
@brockfanning @phillipgardiner @anncorp I finished making changes to this - mostly bringing more in line with UK FAQ page and making the questions a bit more general i.e. less UK specific as thought some of it may be misleading if a country didn't update. I'm not sure who wants to review?

Re. "What browsers can I use to view this website" - I wasn't sure if the answer is still technically correct??

**Comment by brockfanning** [comment]
@LucyGwilliamAdmin Good catch, no we are definitely _not_ working on any IE8 issues. :)

**Review by AnnCorp — APPROVED** [review]

**Pull request merged**